src/imp/noop_imp.js

```diff
@@ -22,7 +29,7 @@ function NoopTracerImp() {
 }
 NoopTracerImp.prototype.setInterface = function (inf) {
 }
-NoopTracerImp.prototype.startSpan = function (nameOrFields, fields) {
```

Comment:
Right, the implementation objects have their parameters pre-normalized so that it's easier to create an OpenTracing implementation (and there's less room for error).